django/contrib/postgres/functions.py

```diff
@@ -3,17 +3,9 @@
 
 class RandomUUID(Func):
     template = 'GEN_RANDOM_UUID()'
-
-    def __init__(self, output_field=None, **extra):
```

Comment:
@sir-sigurd I still hold that we never documented usage of `output_field` as a positional argument for any `Func` subclasses or `Func` itself. In the case of `RandomUUID` at least it shouldn't be an issue as it's not even part of a Django release yet (it was added in 2.0) and it's extremely unlikely someone want's to a use a custom `output_field` for this function.